kubernetes/samples/charts/nginx/samples/tls.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+apiVersion: voyager.appscode.com/v1beta1
+kind: Ingress
+metadata:
+  name: voyager-ssl-1
+  namespace: weblogic-domain1
+  annotations:
+    ingress.appscode.com/type: 'NodePort'
+    ingress.appscode.com/stats: 'true'
+    ingress.appscode.com/affinity: 'cookie'
```

Comment:
those annotations are for Voyager. You need to change it to NGINX related annotation.
kubernetes.io/ingress.class: "nginx"

Reply:
done